File: app/src/main/java/com/wq/editer/Toolbar.java
package com.wq.editer;


import com.wq.editer.button.Button;

public interface Toolbar {
    /**
     * Get button by given button name.
     *
     * @param buttonName Button name.
     * @return
     */
    public Button getButton(String buttonName);

    public void popover(String buttonName, final String params, final String callbackName);

    public void setButtonEnabled(String buttonName, final boolean enabled);

    public void setButtonActivated(String buttonName, final boolean activated);

    public void resetButtonsStatus();
}

File: app/src/main/java/com/wq/editer/Toolbar.kt
package com.wq.editer


import com.wq.editer.button.Button

interface Toolbar {
    /**
     * Get button by given button name.

     * @param buttonName Button name.
     * *
     * @return
     */
//    fun getButton(buttonName: String): Button

    fun popover(buttonName: String, params: String, callbackName: String)

    fun setButtonEnabled(buttonName: String, enabled: Boolean)

    fun setButtonActivated(buttonName: String, activated: Boolean)

    fun resetButtonsStatus()
}

